src/Equinor.ProCoSys.IPO.Infrastructure/Repositories/InvitationRepository.cs

```diff
@@ -31,6 +28,52 @@ public void UpdateCommPkgOnInvitations(string projectName, string commPkgNo, str
             commPkgsToUpdate.ForEach(cp => cp.Description = description);
         }
 
+        public void MoveCommPkg(string fromProject, string toProject, string commPkgNo, string description)
+        {
+            var commPkgsToMove = _context.CommPkgs.Where(cp => cp.ProjectName == fromProject && cp.CommPkgNo == commPkgNo).ToList();
+            var mcPkgsToMove = _context.McPkgs.Where(mc => mc.ProjectName == fromProject && mc.CommPkgNo == commPkgNo).ToList();
+
+            var invitationsToMove =
```

Comment:
won't this move invitations if just one of the commpkgs in the scope has been moved?
Coult have an IPO with 10 commpkgs, and if one of them are moved then this code would move the IPO (which it shouldn't). But I don't know what should happen - they were supposed to discuss it after our meeting yesterday?

Reply:
Thats correct! But I dont see any other options for now. Hopefully they will give us some feedback pretty soon, but it might take weeks.

If not we could  just put everything on hold, if that is an option?

Reply:
I don't know which is better, it messes up the data either way....

Reply:
Maybe add a comment with the "flaw" of doing this now

Reply:
Will add a comment about this in the code

Reply:
Will add in another branch/PR where i handle more sync